modules/microshift-uninstalling-lvms-csi-driver.adoc

```diff
@@ -0,0 +1,53 @@
+// Module included in the following assemblies:
+//
+// * microshift_storage/microshift-storage-plugin-overview.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="microshift-uninstalling-lvms-csi-driver_{context}"]
+= Uninstalling the CSI driver implementation
+
+To uninstall the installed CSI driver implementation, use the following procedure.
+
+.Prerequisites
+
+* {microshift-short} is installed and running.
+* The CSI driver implementation is deployed on the MicroShift cluster.
+
+.Procedure
+
+. Delete the `vg-manager` by running the following command:
++
+[source,terminal]
+----
+$ oc delete -n openshift-storage daemonset.apps/vg-manager
+----
++
+.Example output
+[source,terminal]
+----
+daemonset.apps "vg-manager" deleted
+----
+. Delete the `lvms-operator` by running the following command:
++
+[source,terminal]
+----
+$ oc delete -n openshift-storage deployment.apps/lvms-operator
+----
++
+.Example output
+[source,terminal]
+----
+deployment.apps "lvms-operator" deleted
+----
+. Delete the `topolvm-provisioner` `StorageClass` by running the following command:
```

Comment:
Per some great testing by @dhensel-rh, this operation cannot produce the expected outcome. This is because the daemonset/vg-manager object is owned by another API instance (`lvmclusters.lvm.topolvm.io`). Thus, deleting the daemonset only results in it being redeployed. Instead, users should delete the `lvmclusters.lvm.topolvm.io` instance, which will cause the deletion of the daemonset.

Fix below:

```suggestion
. Delete the `lvmclusters` object by running the following command:
+
[source,terminal]
----
$ oc delete -n openshift-storage lvmclusters.lvm.topolvm.io/lvms
----
+
.Example output
[source,terminal]
----
daemonset.apps "vg-manager" deleted
----
. Delete the `lvms-operator` by running the following command:
+
[source,terminal]
----
$ oc delete -n openshift-storage deployment.apps/lvms-operator
----
+
.Example output
[source,terminal]
----
deployment.apps "lvms-operator" deleted
----
. Delete the `topolvm-provisioner` `StorageClass` by running the following command:
```